packages/modules/viewer-react/src/components/app-ui/providers/DefaultContentGroupProvider.ts

```diff
@@ -55,7 +59,7 @@ export class DefaultContentGroupProvider extends ContentGroupProvider {
       contents: [
         {
           id: "iTwinViewer.UnifiedSelectionViewport",
-          classId: UnifiedSelectionViewportControl,
+          classId: this._syncWithUnifiedSelectionStorage ? IModelViewportControl : UnifiedSelectionViewportControl,
```

Comment:
is the order here correct? should it be reveresed?

Reply:
Yes, it is correct. Before, we used the `viewWithUnifiedSelection` HOC to provide synchronization with unified selection; however, with these changes, the `enableUnifiedSelectionSyncWithIModel` hook is used instead, and so we no longer need to wrap `IModelViewportControl`.